app/src/keymap.c

```diff
@@ -98,50 +97,62 @@ int zmk_keymap_layer_toggle(u8_t layer)
 		return zmk_keymap_layer_deactivate(layer);
 	}
 
-  	return zmk_keymap_layer_activate(layer);
+	return zmk_keymap_layer_activate(layer);
 };
 
-bool is_active_position(u32_t position, u8_t layer)
+bool is_active_position(u32_t position, u8_t current_layer, u32_t layer_state)
```

Comment:
Seems like you can remove the `position` parameter here, since it's not used any more.